zipkin-ui/static/traces.properties

```diff
@@ -6,6 +6,7 @@ traces.duration = Duration (μs) >=
 traces.limit = Limit
 traces.findBtn = Find Traces
 traces.sort = Sort
+traces.searchDisabled = Searching has been disabled via the searchEnabled property.
```

Comment:
@MrGlaucus @gianarb @drolando can you contribute chinese and italian text for this?